broker-cli/proto/broker.proto

```diff
@@ -794,6 +854,117 @@ service OrderService {
       get: "/v1/orders"
     };
   }
+  /**
+   * GetTradeHistory returns all completed and executing orders and all accepted and executed fills.
+   *
+   * ```javascript
+   * const deadline = new Date().setSeconds(new Date().getSeconds() + 5)
+   * const blockOrder = orderService.getTradeHistory({}, { deadline }, (err, res) => {
+   *   if (err) return console.error(err)
+   *   console.log(res)
+   * })
+   * ```
+   *
+   * ```shell
+   * > sparkswap order trade-history
+   * ```
+   *
+   * > The above command will retrieve the trades on the Broker and will print:
+   *
+   * ```javascript
+   * {
+   *   "orders": [
+   *     {
+   *       "orderId": "Ky13eTmuhZY-NIbqIng_in3D",
+   *       "orderStatus": "PLACED",
+   *       "amount": "0.0001000000000000",
+   *       "price": "0.0001000000000000",
+   *       "orderError": "",
+   *       "blockOrderId": "XLDXVDZiXxdifizQ",
+   *       "baseSymbol": "BTC",
+   *       "counterSymbol": "LTC",
+   *       "side": "ASK",
+   *       "dates": {
+   *         "dateCreated": "2019-04-12T18:22:13.111Z",
+   *         "datePlaced": "2019-04-12T18:22:13.121Z",
+   *         "dateExecuting": "",
+   *         "dateCompleted": "",
+   *         "dateRejected": "",
+   *         "dateCancelled": ""
+   *       }
+   *       "fillAmount": "0.0001000000000000",
+   *     },
+   *     {
+   *       "orderId": "U6WQ8Al2lDXqHthXx7UORN0c",
+   *       "orderStatus": "PLACED",
+   *       "amount": "0.0001000000000000",
+   *       "price": "0.0001000000000000",
+   *       "orderError": "",
+   *       "blockOrderId": "XLDXclazEcJjK8Po",
+   *       "baseSymbol": "BTC",
+   *       "counterSymbol": "LTC",
+   *       "side": "ASK",
+   *       "dates": {
+   *         "dateCreated": "2019-04-12T18:22:42.519Z",
+   *         "datePlaced": "2019-04-12T18:22:42.534Z",
+   *         "dateExecuting": "",
+   *         "dateCompleted": "",
+   *         "dateRejected": "",
+   *         "dateCancelled": ""
+   *       }
+   *     }
+   *   ],
+   *   "fills": []
```

Comment:
fills would be good to add to the docs too since they have a different data structure